scripts/run_tests.sh

```diff
@@ -20,7 +20,7 @@ else
     black slack_bolt/ tests/ \
       && pytest \
       && pip install -e ".[adapter]" \
-      && pip install -U pytype \
+      && pip install -U pip setuptools wheel pytype \
```

Comment:
Unrelated to this main change in this PR. Installing pytype often fails due to old versions of setuptools and wheel. Doing this prevents the issue from happening.